From 644ec4d18eb83a0ca826001ebfb7f4782cb8db0f Mon Sep 17 00:00:00 2001
From: Koen Kooi <koen.kooi@linaro.org>
Date: Thu, 19 Jun 2014 13:16:21 +0200
Subject: [PATCH] configure: use pkg-config for libSDL detection

Signed-off-by: Koen Kooi <koen.kooi@linaro.org>
---
 configure.in | 14 ++++++--------
 1 file changed, 6 insertions(+), 8 deletions(-)

diff --git a/configure.in b/configure.in
index e7010e9..cbb536d 100644
--- a/configure.in
+++ b/configure.in
@@ -113,14 +113,12 @@ find_lib()
 }
 
 dnl Check for SDL
-SDL_VERSION=1.2.10
-AC_SUBST(SDL_VERSION)
-AM_PATH_SDL($SDL_VERSION,
-            :,
-           AC_MSG_ERROR([*** SDL version $SDL_VERSION not found!])
-)
-CFLAGS="$CFLAGS $SDL_CFLAGS"
-LIBS="$LIBS $SDL_LIBS"
+PKG_CHECK_MODULES([LIBSDL], [sdl > 1.2.10], [
+  CFLAGS="$CFLAGS $LIBSDL_CFLAGS"
+  LIBS="$LIBS $LIBSDL_LIBS"
+], [
+  AC_MSG_ERROR([$LIBXML_PKG_ERRORS])
+])
 saved_LIBS="$LIBS"
 
 dnl Check command-line options
-- 
1.9.3

